samples/nrf9160/lwm2m_client/overlay-adv-firmware.conf

```diff
@@ -0,0 +1,7 @@
+CONFIG_LWM2M_CLIENT_UTILS=y
+CONFIG_LWM2M_CLIENT_UTILS_ADV_FIRMWARE_UPDATE_OBJ_SUPPORT=y
+CONFIG_LWM2M_CLIENT_UTILS_FIRMWARE_UPDATE_OBJ_SUPPORT=y
+CONFIG_LWM2M_FIRMWARE_UPDATE_OBJ_SUPPORT=n
+CONFIG_FOTA_CLIENT_EVT_DONE_SHEDULE_DFU=n
+# Application version
+CONFIG_MCUBOOT_IMAGE_VERSION="1.0.0"
```

Comment:
```suggestion
# Application version
CONFIG_MCUBOOT_IMAGE_VERSION="1.0.0"
```

Drop these.
Nothing to do with this object.

Reply:
I move this to Prj.conf